src/main/resources/assets/modularforcefieldsystem/recipes/_constants.json

```diff
@@ -99,64 +87,8 @@
     }],
     "ingredient":
     {
-      "item": "ic2:crafting",
-      "data": 2
-    }
-  },
-  {
```

Comment:
Why remove these constants? They can be double defined so long as the conditions are mutually exclusive (otherwise the last valid one will overwrite all previous ones).

```
{
	"name": "OVERCLOCKER",
	"conditions": [{
		"type": "forge:mod_loaded",
		"modid": "ic2"
	}, {
		"type": "forge:not",
		"value": {
			"type": "forge:mod_loaded",
			"modid": "ic2-classic-spmod"
		}
	}],
	"ingredient": {
		"item": "ic2:upgrade",
		"data": 0
	}
},
{
	"name": "OVERCLOCKER",
	"conditions": [{
		"type": "forge:mod_loaded",
		"modid": "ic2-classic-spmod"
	}],
	"ingredient": {
		"item": "ic2:ic2upgrades",
		"data": 0
	}
}
```

This way the recipes don't need to be duplicated just to change the item they're using

Reply:
I was not aware you could duplicate constants. I'll add the constants back then